### PR TITLE
change the `new` to `new-account`,  `set-node` to  `set-node-url` in wallet CLI

### PR DESCRIPTION
**Pull request by qrayven**
# Description of change

changes for better UX:
-> `new` to `new-account`
-> `set-node` `set-node-url`

## Links to any relevant issues

Part of: #389 

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: sdk/CHANGELOG.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Rename `WalletOptions::build_manager` to `build`;
 - `PeerDto` renamed to `PeerResponse`, `ReceiptDto` to `ReceiptResponse`, `LedgerInclusionStateDto` to `LedgerInclusionState`, `HeartbeatDto` to `Heartbeat`, `MetricsDto` tp `Metrics`, `GossipDto` to `Gossip`, `RelationDto` to `Relation`;
 - Default number of workers for nonce `Miner` changed from `1` to `num_cpu::get()`;
+- Rename `new` to `new-account` and `set-node` to `set-node-url` in wallet CLI;
```
Comment (thibault-martinez) [inline sdk/CHANGELOG.md]: This changelog is just for the library, remove

File: cli/src/command/wallet.rs [hunk 2]
```diff
@@ -72,8 +72,8 @@ pub enum WalletCommand {
         /// Path of the to be restored stronghold backup file.
         backup_path: String,
     },
-    /// Set the node to use.
-    SetNode {
+    /// Set the URL of the node to use.
+    SetNodeURL {
```
Comment (thibault-martinez) [inline cli/src/command/wallet.rs]:
```suggestion
    SetNodeUrl {
```

**Comment by Alex6323** [comment]
I am okay with this ... but .. can you fix the PR title :joy: and mark it a breaking change?

File: cli/src/command/wallet.rs (hunk 2 above)
Comment (qrayven) [reply cli/src/command/wallet.rs]: fixed

File: sdk/CHANGELOG.md (hunk 1 above)
Comment (qrayven) [reply sdk/CHANGELOG.md]: fixed

**Comment by qrayven** [comment]
> I am okay with this ... but .. can you fix the PR title joy and mark it a breaking change?

what a typo, thanks :D

File: cli/src/wallet.rs [hunk 3]
```diff
@@ -42,11 +42,11 @@ pub async fn new_wallet(cli: WalletCli) -> Result<(Option<Wallet>, Option<String
                 migrate_stronghold_snapshot_v2_to_v3_command(path).await?;
                 return Ok((None, None));
             }
-            WalletCommand::New { alias } => {
+            WalletCommand::NewAccount { alias } => {
                 let (wallet, account) = new_command(storage_path, snapshot_path, alias).await?;
```
Comment (thibault-martinez) [inline cli/src/wallet.rs]: I would adapt the name of the methods as well for consistency

Comment (qrayven) [reply cli/src/wallet.rs]: fixed

File: cli/src/wallet.rs [hunk 4]
```diff
@@ -42,11 +42,11 @@ pub async fn new_wallet(cli: WalletCli) -> Result<(Option<Wallet>, Option<String
...
             }
-            WalletCommand::New { alias } => {
-                let (wallet, account) = new_command(storage_path, snapshot_path, alias).await?;
+            WalletCommand::NewAccount { alias } => {
+                let (wallet, account) = new_account_command(storage_path, snapshot_path, alias).await?;
                 (Some(wallet), Some(account))
             }
-            WalletCommand::SetNode { url } => {
+            WalletCommand::SetNodeUrl { url } => {
                 let wallet = set_node_command(storage_path, snapshot_path, url).await?;
```
Comment (thibault-martinez) [inline cli/src/wallet.rs]: I was assuming you'd change this one as well sorry, should have left another comment

**Review by DaughterOfMars — DISMISSED** [review]

File: cli/src/wallet.rs (hunk 4 above)
Comment (qrayven) [reply cli/src/wallet.rs]: Sorry, I actually shoul've change it. Fixed

**Review by thibault-martinez — DISMISSED** [review]

File: cli/README.md [hunk 5]
```diff
@@ -29,10 +29,10 @@ If you already created an account, you can just run the CLI without args to get
 ./wallet
 ```
 
-Alternatively, you can select an existing account by it's alias, to use with the `account` command:
+Alternatively, you can select an existing account by it's alias:
```
Comment (Alex6323) [inline cli/README.md]:
```suggestion
Alternatively, you can select an existing account by its alias:
```
sorry, once I see a typo I cannot resist from mentioning it :trollface:

**Review by thibault-martinez — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**